### PR TITLE
Minesweeper game is developed

### LINKED ISSUE #724: Kitchen Sink : Development of a Minesweeper Game
### Describe the feature

**Describe the feature** -
 I would like to develop the Minesweeper game and add it to this repository.

- The minesweeper game follows standard rules, with a grid of your choice (max: 18 x 18).
- Implement a random generator for mines and blank squares on the grid area.
- Ensure the game is winnable and has an outcome every game.
- Allow users to check the grids and specify them whether its a mine or not through small image.
- Add a field to specify the number of mines to be spawned in each game with it's counter during game.
- Integrate visually appealing graphics and user-friendly controls for this minesweeper game.

Could you please assign me this issue?

### Additional information

- [X] Would you be willing to help implement this feature?

### PR DESCRIPTION
**Pull request by neelesh133**
resolves #724 

**Description**
I have developed and tested the Minesweeper game. It is fully functional and ready for review.

**Game Features**

- A minesweeper game with its main goal to clear a rectangular board containing hidden "mines" or bombs without detonating any of them.
- Allow users to choose the difficulty level of the game, such as 1,2,3....7. According to which grid size and number of mines 
changes.
-  If the user opens a cell with a bomb, display a loss message in a toast notification.
- Allow users to reset game with help of a reset button.

**Below are some Screenshots and video of the game-**

Screenshot 2023-10-14 002021
Screenshot 2023-10-14 002058

https://github.com/aidenybai/million/assets/119878088/07dca57a-411b-474a-9f94-759aafec55de

<h3>I am in the process of creating a pull request for this feature. Your feedback is welcome!</h3>
<hr/>
**Status**

- [x] Code changes have been tested against prettier, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by tobySolutions** [comment]
> [CLA assistant check](https://cla-assistant.io/aidenybai/million?pullRequest=739) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aidenybai/million?pullRequest=739) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aidenybai/million?pullRequest=739) it.

Hey there @neelesh133, can you maybe sign the CLA?

File: packages/kitchen-sink/src/examples/minesweeper.jsx [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+        />
+      </label>
+      <Minesweeper level={level} />
+    </div>
+  );
+}
+
+//Minesweeper.jsx
+
+function Minesweeper({ level }) {
```
Comment (tobySolutions) [inline packages/kitchen-sink/src/examples/minesweeper.jsx:145]: Hey there, did you try using a `block` here to wrap this component maybe?

**Comment by neelesh133** [comment]
> Hey there @neelesh133, can you maybe sign the CLA?

@tobySolutions i have signed the CLA.

**Thank You!**

**Comment by neelesh133** [comment]
Could you review my PR. Thank You.

**Comment by tobySolutions** [comment]
> @tobySolutions Could you review my PR. Thank You.

Awesome! Checking it out now @neelesh133. Thanks!

**Pull request merged**